README.md

```diff
@@ -12,11 +12,9 @@ and our [website](https://dockstring.github.io/):
 
 ## Installation
 
-Currently this package is **only supported on Linux**,
-and we do not plan to add support for Mac/Windows.
-See [this issue](https://github.com/dockstring/dockstring/issues/141)
-for more information on why this is
-(feel free to re-open it if you want to help us provide Mac/Windows support).
+This package is primarily intended for Linux, but we have some support for Mac.
```

Comment:
I would change

`However, the scores from the Mac version does not match the Linux version,
so we encourage the use of the Linux version whenever possible.`

for 

`Please note that the scores from the Mac version do not always perfectly match the Linux version, so we encourage the use of the Linux version whenever possible.`